docs/closure-compiler.md

```diff
@@ -46,21 +46,20 @@ checked under the `optimization` option.
 
 Furthermore, you may want to check the `Pretty Print` option if you're debugging the output code.
 
-Additionally, if you use getter/setter properties within your code, add the following in between
-the `// ==ClosureCompiler==` / `// ==/ClosureCompiler==` comment block:
+Additionally, if you use getter/setter properties within your code, add `// @language_out ECMASCRIPT5` in between the ClosureCompiler comment block, like:
 
-```js
+```
+// ==ClosureCompiler==
 // @language_out ECMASCRIPT5
+// ...Other configurations
+// ==/ClosureCompiler==
 ```
 
 This will tell closure to compile your code to ES5 (the default is ES3 which doesn't support
 accessor properties).
 
-Finally, you can make the permalinks to closure code a bit nicer by using a link shortener, such as
```

Comment:
I removed this because I remember in the tutorial it said:

> If you want the Closure Compiler service to serve the compressed file, choose a name for the output file using the @output_file_name parameter at the top of the input field. Note that the default for the output file name is default.js but you should change it to a more meaningful name for your project. Also note that the Closure Compiler service hosts the output file on its servers for **an hour**.

Also feel we are selling a lot at the same time but this is relatively less important...